server/constants/index.js

```diff
@@ -6,10 +6,13 @@ import {
   ALL_FILTERS_ACTIVE,
   FILTER_TO_MONGO_VALUE_MAP,
   INCLUSION_EXCLUSION_CRITERIA_FORM,
+  SOCIO_DEMOGRAPHICS_FORM,
```

Comment:
sociodemographics is 1 word 😄 

```suggestion
  SOCIODEMOGRAPHICS_FORM,
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/280/commits/81cad2058d15f6a15a126abcecbb091090e31058